### DISCUSSION TITLE
need to use RFM95w module instead of standard lora devolopment kit

### DISCUSSION THREAD

**Discussion by sridhar2rok** [Lora-net/LoRaMac-node#1161] [Q&A] [answered]
Which files/configurations are needed to be changed to interface the RFM95W module with this library? As the library here uses ant switch, the busy pin. This is not present in the RFM95w module.

**Comment by mluis1** [comment, marked as answer]
If I'm not mistaken the RFM95W is equivalent to SX1276 Semtech radio. Please select one of the possible combinations that use SX1276 radio.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.
